semantic_router/encoders/huggingface.py

```diff
@@ -212,19 +212,17 @@ def __call__(self, docs: List[str]) -> List[List[float]]:
             ValueError: If no embeddings are returned for a document.
         """
 
-        batch_size=50
+        batch_size = 50
```

Comment:
ValueError: No embeddings returned for batch. Error: Query failed with status 413: {"error":"batch size 50 > maximum allowed batch size 32","error_type":"Validation"}

Reply:
Hi @joaomsimoes what HuggingFace TEI model were you using when you encountered this error?